cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -608,17 +608,22 @@ shotgun_install_dir_to_path() {
     local _shell="$5"
 
     if [ -n "${HOME:-}" ]; then
-        local _target
+        local _found=false
         local _home
 
         for _rcfile_relative in $_rcfiles; do
             _home="$(print_home_for_script "$_rcfile_relative")"
             local _rcfile_abs="$_home/$_rcfile_relative"
 
             if [ -f "$_rcfile_abs" ]; then
+                _found=true
                 add_install_dir_to_path "$_install_dir_expr" "$_env_script_path" "$_env_script_path_expr" "$_rcfile_relative" "$_shell"
             fi
         done
+
+	    if [ "$_found" = false ]; then
+            add_install_dir_to_path "$_install_dir_expr" "$_env_script_path" "$_env_script_path_expr" "$(echo "$_rcfiles" | awk '{ print $1 }')" "$_shell"
```

Comment:
What does this extra awk accomplish?

Reply:
Oops - nothing, that was left over from the check I pasted from `add_install_dir_to_path` before I checked and found that `var=bool` is dash/POSIX compliant.